docs/reference/configuration/resource-configuration-and-ha.md

```diff
@@ -144,6 +144,8 @@ packages:
               path: postgresql.password
 ```
 
+When running Keycloak on new Kernels 6.12+, it is also necessary to specify the `KEYCLOAK_KERNEL_6_12_MODE` setting and set it `true`. This is a temporary workaround for [this](https://github.com/keycloak/keycloak/issues/36609) Keycloak issue. Once addressed, this setting will be removed.
```

Comment:
We may want to adjust this note to provide an example of the bundle `overrides` block that would be necessary. For the dev/demo bundles this value is setup but for an end user making their own bundle they would need to set up overrides to match (with either values or variables) in order to have this set properly.

Reply:
Added a meaningful example.